bind-all.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @module 101/bind-all
+ */
+
+var isFunction = require('./is-function');
+var keysIn = require('./keys-in');
+/**
+ * Bind a passed object methods.
+ * If methods name to bing are not specified, all the object methods are binded
+ * @function module:101/bind-all
+ *
+ * @param {object} object - object to bind
+ * @param {array|string} [methods] - array or space-separated string containing the names of the methods to bind
+ * @return {object} the binded object
+ */
+module.exports = bindAll;
+
+function bindAll (object, methods) {
+  var keys;
+  if (methods) {
+    // if we passed an array, set keys to the passed array
+    if (Array.isArray(methods)) {
+      keys = methods;
+    } else if (typeof methods === 'string') {
+      // if a string was paased, split it by spaces
+      keys = methods.split(/,\s|[,\s]/);
```

Comment:
> it doesn't seem very natural to me

yop to me too.

I would be fine with just excepting an array and no string at all. I would even skip `bindAll(obj, 'func')` I think as @tjmehta said it's a trivial operation to split the string or check if it's an array.